<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriveMode</class>
 <widget class="QDialog" name="DriveMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>321</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Drive Vehicle</string>
      </property>
      <widget class="QPushButton" name="LeftForward_Button">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>80</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    qproperty-icon: url(:/images/images/Button1.PNG);
    qproperty-iconSize: 70px 90px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Forward_Button">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>80</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    qproperty-icon: url(:/images/images/Button2.PNG);
    qproperty-iconSize: 70px 90px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="RightForward_Button">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>80</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    qproperty-icon: url(:/images/images/Button3.PNG);
    qproperty-iconSize: 70px 90px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Left_Button">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>80</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    qproperty-icon: url(:/images/images/Button4.PNG);
    qproperty-iconSize: 70px 90px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Stop_Button">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>150</y>
         <width>80</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    qproperty-icon: url(:/images/images/Button5.PNG);
    qproperty-iconSize: 70px 90px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Right_Button">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>150</y>
         <width>80</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    qproperty-icon: url(:/images/images/Button6.PNG);
    qproperty-iconSize: 70px 90px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="LeftBackward_Button">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>80</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    qproperty-icon: url(:/images/images/Button7.PNG);
    qproperty-iconSize: 70px 90px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Backward_Button">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>260</y>
         <width>80</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    qproperty-icon: url(:/images/images/Button8.PNG);
    qproperty-iconSize: 70px 90px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="RightBackward_Button">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>260</y>
         <width>80</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    qproperty-icon: url(:/images/images/Button9.PNG);
    qproperty-iconSize: 70px 90px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="LeftRotate_Button">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>370</y>
         <width>80</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    qproperty-icon: url(:/images/images/Button10.PNG);
    qproperty-iconSize: 70px 90px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="RightRotate_Button">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>370</y>
         <width>80</width>
         <height>100</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    qproperty-icon: url(:/images/images/Button11.PNG);
    qproperty-iconSize: 70px 90px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vehicle Control</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Change Vehicle Speed</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Gripper Control</string>
   </property>
   <widget class="QSlider" name="GripperSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>160</number>
    </property>
    <property name="value">
     <number>120</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Linear Guide Control</string>
   </property>
   <widget class="QPushButton" name="UpButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>GoUp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DownButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>GoDown</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>460</y>
     <width>201</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Recording and Playback</string>
   </property>
   <widget class="QPushButton" name="StartRecordBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Recording</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopRecordBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Recording</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteRecordBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Recording</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayBackBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PlayBack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveRecordBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayBackSavedBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PlayBack Saved</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
